src/main/kotlin/com/learnspigot/bot/util/Server.kt

```diff
@@ -0,0 +1,25 @@
+package com.learnspigot.bot.util
+
+import com.learnspigot.bot.Bot
+import com.learnspigot.bot.Environment
+import net.dv8tion.jda.api.entities.emoji.Emoji
+
+object Server {
```

Comment:
I like the idea of what's done with this class but I feel like it should reside in the existing `Environment` class.

Just add these values in it imo as that best suits and describes what these values are - I don't see the relevance of `Server`.

Reply:
K great

Reply:
This isn't worth the debate since we clearly both just see it differently